lib/compilers/assembly.ts

```diff
@@ -51,7 +51,7 @@ export class AssemblyCompiler extends BaseCompiler {
         return [];
     }
 
-    override getArgumentParser() {
+    override getArgumentParserCls() {
```

Comment:
What triggered this rename? And why "Cls" and not "Class"?

Reply:
This doesn't actually return a parser, but the *class* object of a parser. Later on class methods are invoked on the returned object with '.' syntax.  That was very confusing, and only after realizing it was I able to fix the return hint from any to a proper type.

I'm fine with renaming to *Class if you prefer.

Reply:
Yes I see now that *Class is the CE naming convention. Pushed a re-rename.